MAUI/Popup/styles.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: Styles in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Styles support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+--- 
+
+# Styles in .NET MAUI Popup (SfPopup)
+
+The SfPopup applies style to all of its elements by using the `SfPopup.PopupStyle` property.
+
+## Styling popup header
+
+The SfPopup allows customizing the header elements with various header customizations as follows:
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderBackground </td>
+<td> Gets or sets the background color for the header.</td>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderFontAttribute </td>
+<td> Gets or sets the font attribute for the header title.</td>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderFontFamily </td>
+<td> Gets or sets the font style for the header title.</td>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderFontSize </td>
+<td> Gets or sets the font size for the header title.</td>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderTextAlignment </td>
+<td> Gets or sets the text alignment for the header.</td>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderTextColor </td>
+<td> Gets or sets the text color to be applied for the header title.</td>
+</tr>
+</table>
+
+Refer to the following code example for customizing the header elements.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines=" 3 4 5 6 7 8" %}
+<sfPopup:SfPopup x:Name="popup" >
+    <sfPopup:SfPopup.PopupStyle>
+        <sfPopup:PopupStyle HeaderBackground="DimGray"
+                            HeaderFontAttribute="Bold"
+                            HeaderFontFamily="Roboto-Medium"
+                            HeaderFontSize="25"
+                            HeaderTextAlignment="Center"
+                            HeaderTextColor="White"/>
+    </sfPopup:SfPopup.PopupStyle>
+</sfPopup:SfPopup>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines=" 4 5 6 7 8 9" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.PopupStyle.HeaderBackground = Color.FromRgb(105,105,105);
+    popup.PopupStyle.HeaderFontAttribute = FontAttributes.Bold;
+    popup.PopupStyle.HeaderFontFamily = "Roboto-Medium";
+    popup.PopupStyle.HeaderFontSize = 25;
+    popup.PopupStyle.HeaderTextAlignment = TextAlignment.Center;
+    popup.PopupStyle.HeaderTextColor = Color.White;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with header customization](/Images/styles/maui-popup-with-header-customization.png)
+
+## Styling popup footer
+
+The SfPopup allows customizing the footer elements with various footer customizations as follows:
```

Comment:
The SfPopup allows customizing the footer appearance using the below properties.

Reply:
modified